### PR TITLE
SW-2608 Replace Google analytics global site tag with Google tag manager

### PR DESCRIPTION
**Pull request by alexTerraformation**
Move cookie script to GTM
Support staging and prod

### CODE REVIEW

**Review by constanzauanini — APPROVED** [review]
lgtm!

File: public/index.html [hunk 1]
```diff
@@ -1,18 +1,42 @@
...
+          window,
+          document,
+          'script',
+          'dataLayer',
+          'GTM-TNX8NWM',
+          '&gtm_auth=oxl4mxgvnvET8ov_75EuYA&gtm_preview=env-1&gtm_cookies_win=x'
+        );
+      } else {
+        // Track non-staging/prod environments
+        // gtm(window, document, 'script', 'dataLayer', 'GTM-TNX8NWM', '&gtm_auth=1591mjZEVL_pulZkOX4gWg&gtm_preview=env-11&gtm_cookies_win=x');
```
Comment (karthikbtf) [inline public/index.html:36]: remove?

Comment (alexTerraformation) [reply public/index.html:36]: I left it for testing (or if we eventually add a third stage to our dev pipeline).  Any objections to leaving it for those reasons?

Comment (karthikbtf) [reply public/index.html:36]: No objections, we can certainly keep it. Might be good to clarify the comment .. "Uncomment this section to test non-staging/prod environnents" or something like that..

**Review by karthikbtf — APPROVED** [review]
🎉

**Pull request merged**